src/simulator_worker/utils.py

```diff
@@ -31,13 +31,27 @@
 logger = logging.getLogger(__name__)
 
 
+def get_port_index(asset: esdl.Asset, porttype: Type) -> int:
```

Comment:
Type annotation for `porttype` may be stricter: `Type[esdl.InPort | esdl. OutPort]`

Reply:
added type notation